sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-CSGD.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSGD.`, () => {
+  it(
+    "Should determine CSGD as eligible when total assessed need is greater than 0 " +
+      ", total eligible dependant is atleast 1 and total family income is less than the " +
+      "threshold for a married student.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+      assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+        YesNoOptions.Yes;
+      assessmentConsolidatedData.studentDataDependantstatus = "independant";
+      assessmentConsolidatedData.studentDataDependants = [
+        {
+          age: null,
+          fullName: "Jack",
+          dateOfBirth: "2023-10-29",
+          validDependent: 0,
+          declaredOnTaxes: YesNoOptions.No,
+          isavalidDependant: true,
```

Comment:
Here,  also not part of this ticket, it should be "isAValidDependant".

Reply:
https://github.com/bcgov/SIMS/pull/2506#discussion_r1393415242